### PR TITLE
RBAC in the flow of the Authorities

### PR DESCRIPTION
**Pull request by garcia-alejandro**
# Description
* The RBAC queries were added to get the information
* Fixed issue on the NFT creation when the Chain doesn't have any NFT
## Ticket
* #55

### CODE REVIEW

File: src/services/polkadot-pallets/marketplaceApi.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import BasePolkadotApi from '~/services/basePolkadotApi'
+import { Rbac } from '../polkadot-pallets'
 // import markets from '~/services/const/allMarkets'
 // import participants from '~/services/const/participants'
 // import applicants from '~/services/const/applicants'
 class MarketplaceApi extends BasePolkadotApi {
   constructor (polkadotApi, notify) {
     super(polkadotApi, 'gatedMarketplace', notify)
+    this.RBAC = new Rbac(polkadotApi)
+    this.palletId = '0x5d208df71427902157988a56e7527474a7dce921f9d5efa159dcfc849289a53c'
```
Comment (jmgayosso) [inline src/services/polkadot-pallets/marketplaceApi.js]: Can we get this value dynamically?

File: src/pages/marketplace/details-market.vue [hunk 2]
```diff
@@ -149,7 +149,8 @@ export default {
       try {
         this.showLoading()
         const market = await this.$store.$marketplaceApi.getMarketplaceById({ marketId: this.marketId })
-        const authorities = await this.$store.$marketplaceApi.getAuthoritiesByMarketplace({ marketId: this.marketId })
+        const palletId = this.$store.$marketplaceApi.palletId
+        const authorities = await this.$store.$marketplaceApi.getAuthoritiesByMarketplace({ palletId, marketId: this.marketId })
```
Comment (jmgayosso) [inline src/pages/marketplace/details-market.vue]: $marketPlaceApi should handler palletId automatically

File: src/services/polkadot-pallets/marketplaceApi.js (hunk 1 above)
Comment (garcia-alejandro) [reply src/services/polkadot-pallets/marketplaceApi.js]: Yes, I think so, there is a way to get the pallet ID. I'll work on it. Thanks!

File: src/pages/marketplace/details-market.vue (hunk 2 above)
Comment (garcia-alejandro) [reply src/pages/marketplace/details-market.vue]: Got it, I'm going to change this. 👍🏼

**Review by jmgayosso — APPROVED** [review]

**Pull request merged**